packages/x-date-pickers/src/internals/models/props/toolbar.ts

```diff
@@ -25,7 +30,7 @@ export interface BaseToolbarProps<TValue, TView extends DateOrTimeView>
   toggleMobileKeyboardView?: () => void;
 }
 
-export interface ExportedBaseToolbarProps {
+export interface ExportedBaseToolbarProps extends Pick<BaseToolbarProps<any, any>, 'hidden'> {
```

Comment:
Why not moving the `hidden` prop here ?
Usually we have `interface XXXProps extends ExportedXXXProps` not the other way around.

Reply:
Good point. Will fix this case, I thought that something seems strange props wise for this component... 🙈